crates/sui-types/src/lib.rs

```diff
@@ -132,6 +135,18 @@ built_in_ids! {
     SUI_DENY_LIST_ADDRESS / SUI_DENY_LIST_OBJECT_ID = 0x403;
 }
 
+/// The set of well-known system (Move) objects
+pub static KNOWN_SYSTEM_OBJECTS: Lazy<BTreeSet<ObjectID>> = Lazy::new(|| {
```

Comment:
I do this a lot too... but I've wondered for small set if it is faster to have `&[ObjectID]` with a `find`?

Reply:
🤷